pkgs/top-level/emacs-packages.nix

```diff
@@ -71,7 +76,12 @@ in makeScope pkgs'.newScope (self: makeOverridable ({
   // manualPackages // { inherit manualPackages; }
   // {
 
-    inherit emacs;
+    # Propagate overriden scope
+    emacs = emacs'.overrideAttrs(old: {
+      passthru = old.passthru // {
+        pkgs = dontRecurseIntoAttrs self;
+      };
```

Comment:
``` nix
old.passthru or {} //
```

This commit seems to have broken my nix-doom-emacs setup.
The patch above "fixes" the issue.

Reply:
This also breaks the emacs-overlay.

Reply:
It doesn't break the overlay, at least not generally speaking.
Probably for doom which uses this code, but that's already fixed by now.

Reply:
Sorry, you're right. I misdiagnosed the problem.

This breaks `emacsMacport`. (I assume it's fixed in #133260.)